XCUITests/TrackingProtectionTests.swift

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class TrackingProtectionTests: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+
+    override func tearDown() {
+        navigator = nil
+        app = nil
+        super.tearDown()
+    }
+    
+    func testTrackingProtection() {
+        navigator.goto(SettingsScreen)
+        let appsettingstableviewcontrollerTableviewTable = app.tables["AppSettingsTableViewController.tableView"]
+        while app.staticTexts["Tracking Protection"].exists == false {
```

Comment:
nit: `== false` to `!app`

Reply:
BTW, this block is cool, I didn't know an easy way to get XCUITest to see table view rows that were offscreen